node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -100,7 +101,11 @@ impl NodeConfigurator<BootstrapperConfig> for NodeConfiguratorStandardUnprivileg
             &mut unprivileged_config,
             Some(persistent_config.as_mut()),
         )?;
-        configure_database(&unprivileged_config, persistent_config.as_mut())?;
+        configure_database(
```

Comment:
Rather than modifying the signature of this method for a special case, can you move the call to set_neighbors_in_zero_hop_if_supplied up into this method?